tutorials/clustering.md

```diff
@@ -167,7 +167,7 @@ You can use Timescale's `distributed_exec` function to perform this action on
 the data nodes:
 ```sql
 CREATE ROLE testuser WITH LOGIN PASSWORD 'testpass';
-SELECT * FROM distributed_exec($$ CREATE ROLE testuser WITH LOGIN PASSWORD 'testpass' $$);
+CALL distributed_exec($$ CREATE ROLE testuser WITH LOGIN PASSWORD 'testpass' $$);
```

Comment:
@k-rus other PR might have handled this as well..

Reply:
Looks like this one wasn't in Ruslan's PR.